client/src/index.js

```diff
@@ -41,23 +42,28 @@ const Routes = () => {
             </Menu.Item>
           </Link>
 
-          <Link className="nav-link" to="/sign-up-user">
-            <Menu.Item name="editorials" active={true}>
-              User Sign Up
-            </Menu.Item>
-          </Link>
-
-          <Link className="nav-link" to="/login">
-            <Menu.Item name="editorials" active={true}>
-              Login
-            </Menu.Item>
-          </Link>
+          {isLoggedIn() ? null : (
+            <Link className="nav-link" to="/sign-up-user">
+              <Menu.Item name="editorials" active={true}>
+                User Sign Up
+              </Menu.Item>
+            </Link>
+          )}
+          {isLoggedIn() ? null : (
+            <Link className="nav-link" to="/login">
+              <Menu.Item name="editorials" active={true}>
+                Login
+              </Menu.Item>
+            </Link>
+          )}
 
-          <Link className="nav-link" to="/sign-up-interpreter">
-            <Menu.Item name="editorials" active={true}>
-              Become Interpreter
-            </Menu.Item>
-          </Link>
+          {isLoggedIn() ? null : (
+            <Link className="nav-link" to="/sign-up-interpreter">
+              <Menu.Item name="editorials" active={true}>
+                Become Interpreter
+              </Menu.Item>
+            </Link>
+          )}
```

Comment:
Can you group all these condition into one? It would make the code more maintainable and easier to read :)